.github/workflows/lint.yaml

```diff
@@ -0,0 +1,68 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+name: Lint
+run-name: ${{ github.actor }} is linting fabric-samples
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - release-1.4
+      - release-2.2
+  pull_request:
+    branches:
+      - main
+      - release-1.4
+      - release-2.2
+
+env:
+  GO_VER: 1.18.3
+  NODE_VER: 16.x
+  JAVA_VER: 11.x
+
+jobs:
+  go:
+    runs-on: ubuntu-latest
```

Comment:
Wonder if this should be fixed at the lowest level of os that we support.. namely 20.04... 
Avoid the issue we saw earlier where the wrong level c runtime was imported